public/js/Math.extensions.js

```diff
@@ -1,23 +1,24 @@
-function polarToCartesian(angle, length, offset) {
+window.polarToCartesian = function (angle, length, offset) {
```

Comment:
file is loaded as script so top level functions are automatically attached to window. here, I'm making it explicit though to appease the linter